<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <widget class="QLineEdit" name="searchBar">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>-9</y>
         <width>271</width>
         <height>51</height>
        </rect>
       </property>
      </widget>
      <widget class="QTabWidget" name="tabWidget">
       <property name="geometry">
        <rect>
         <x>-6</x>
         <y>49</y>
         <width>621</width>
         <height>511</height>
        </rect>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="currentIndex">
        <number>2</number>
       </property>
       <widget class="QWidget" name="Movies">
        <attribute name="title">
         <string>Movies</string>
        </attribute>
        <widget class="QListView" name="moviesView">
         <property name="geometry">
          <rect>
           <x>5</x>
           <y>1</y>
           <width>611</width>
           <height>471</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="Webseries">
        <attribute name="title">
         <string>Webseries</string>
        </attribute>
        <widget class="QListView" name="seriesView">
         <property name="geometry">
          <rect>
           <x>5</x>
           <y>10</y>
           <width>641</width>
           <height>481</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="Songs">
        <attribute name="title">
         <string>Songs</string>
        </attribute>
        <widget class="QListView" name="songsView">
         <property name="geometry">
          <rect>
           <x>5</x>
           <y>10</y>
           <width>671</width>
           <height>491</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QComboBox" name="categoryFilter">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>0</y>
         <width>111</width>
         <height>41</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>All</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>South</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Hindi</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>English</string>
        </property>
       </item>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
